website/pages/en/index.js

```diff
@@ -148,38 +134,108 @@ const GetStartedButton = ({ href }) => (
 )
 
 const BuildingBlocks = props => (
-  <BuildingBlocksContainer>
-    <EmptyStateCard
-      title="aragonOS"
-      text="Solidity framework for governance"
+  <BuildingBlocksContainer className="white-section">
+    <BlocksCards
+      title="AragonOS"
+      text={<p>Solidity framework for governance</p>}
       actionButton={() => <GetStartedButton href={docUrl('aragonos-intro')}/>}
-      icon={() => <IconSettings /> }
+      icon={() => <img src={siteConfig.baseUrl + siteConfig.os} />}
     />
-    <EmptyStateCard
-      title="aragonAPI"
-      text="Easily interact with your dapp's state"
+    <BlocksCards
+      title="AragonAPI"
+      text={<p>Easily interact with your dapp's state</p>}
```

Comment:
I hate GitHub’s highlighting syntax for apostrophes, I did find that using `’` it looks ok.